aopy/analysis.py

```diff
@@ -74,19 +74,21 @@ def factor_analysis_dimensionality_score(data_in, dimensions, nfold, maxiter=100
 
 
 # These functions are for curve fitting and getting modulation depth and preferred direction from firing rates
-def func(target, b1, b2, b3):
+def tuning_fit_func(target, b1, b2, b3):
     '''
-
+    Tuning function to be used in :func:`aopy.analysis.run_curvefitting`
     Args:
         target (int) center out task target index ( takes values from 0 to 7)
         b1, b2, b3 : parameters used for curve fitting
 
     .. math::
-    
+
         b1 * cos(\\theta) + b2 * sin(\\theta) + b3
 
     Returns: result from above equation
 
+    See Also: :func:`aopy.analysis.run_curvefitting` , :func:`aopy.analysis.get_preferred_direction`, :func:`aopy.analysis.get_modulation_depth`
+
```

Comment:
I think an example would be really helpful. That's also a benefit of having a unit test, so we can see how you are supposed to call the functions